recipes/ingrid/setup.py.patch

```diff
@@ -0,0 +1,11 @@
+--- setup.py.orig	2021-06-04 13:34:15.796271806 -0700
++++ setup.py	2021-06-04 13:34:23.408238740 -0700
+@@ -20,8 +20,6 @@
+ 
+ setup(
+     name='INGRID',
+-    version_format='{tag}',
+-    setup_requires=['setuptools-git-version'],
```

Comment:
The version isn't being set. This is probably the easiest workaround though ideally the upstream package should use setuptools_scm instead:
```suggestion
-    setup_requires=['setuptools-git-version'],
+    version=os.environ["PKG_VERSION"],
```

Reply:
I will put this in the patch once a decision is made about how to handle the `tk` pip requirement.